ksql-cli/src/main/java/io/confluent/ksql/cli/console/cmd/History.java

```diff
@@ -31,18 +36,13 @@ public String getName() {
   }
 
   @Override
-  public void printHelp() {
-    console.writer().println(
-        "history:");
-    console.writer().println(
-        "\tShow previous lines entered during the current CLI session. You can"
-            + " use up and down arrow keys to view previous lines."
-    );
+  public String getHelpMessage() {
+    return HELP;
   }
 
   @Override
-  public void execute(final String commandStrippedLine) {
-    console.printHistory();
-    console.flush();
+  public void execute(final List<String> args, final PrintWriter terminal) {
+    CliCmdUtil.ensureArgCountBounds(args, 0, 0, () -> HELP);
+    historyPrinter.run();
```

Comment:
something here seems off to me, we're passing in `terminal`, but we're not printing to that one when we run the history. is there any way to resolve this, or is this intentional?

Reply:
Small steps.  There are definitely other improvements that can be made to these classes, but one thing at a time ;)

Reply:
But.. now that you're raised it... I'll fix this too! ;)